sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -1128,15 +1128,16 @@ def _add_argparse_args(cls, parser):
         type=str,
         help='GCE minimum CPU platform. Default is determined by GCP.')
     parser.add_argument(
-        '--state_cache_size',
-        '--state_cache_size_mb',
-        dest='state_cache_size',
+        '--max_cache_memory_usage_mb',
+        dest='max_cache_memory_usage_mb',
         type=int,
         default=None,
```

Comment:
Any concerns to define the 100mb default here?

Reply:
Current flow: If it is None here, it gives us an opportunity to look in `--experiements` for `state_cache_size`. 

If the value is defined here as 100 MB, and if the user passes `--experiments=state_cache_size`, we should override 100 MB for the `--experiments=state_cache_size`. 

I don't see any concerns of setting default here. might need to change some code though